lib/config/defaults.js

```diff
@@ -920,8 +921,16 @@ const applyOutputDefaults = (
 	);
 	F(output.environment, "forOf", () => tp && optimistic(tp.forOf));
 	F(output.environment, "bigIntLiteral", () => tp && tp.bigIntLiteral);
-	F(output.environment, "dynamicImport", () => tp && tp.dynamicImport);
-	F(output.environment, "module", () => tp && tp.module);
+	F(
+		output.environment,
+		"dynamicImport",
+		() => tp && conditionallyOptimistic(tp.dynamicImport, output.module)
```

Comment:
```suggestion
		() => conditionallyOptimistic(tp && tp.dynamicImport, output.module)
```

I wonder why it's not that way? Shouldn't be default to `output.module` when there are no target properties?

Reply:
yeah, lets do this way.